src/wp-includes/media.php

```diff
@@ -1520,8 +1520,8 @@ function wp_get_attachment_image_sizes( $attachment_id, $size = 'medium', $image
  *
  * @param string|int[] $size          Image size. Accepts any registered image size name, or an array of
  *                                    width and height values in pixels (in that order).
- * @param string       $image_src     Optional. The URL to the image file. Default null.
- * @param array        $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'.
+ * @param string|null  $image_src     Optional. The URL to the image file. Default null.
+ * @param array|null   $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'.
```

Comment:
```suggestion
 * @param array|null   $image_meta    Optional. The image meta data as returned by 'wp_get_attachment_metadata()'. Default null.
```

Add `Default null.` in document.

Reply:
@mukeshpanchal27 `Default null.` already there
Ref: https://github.com/dilipbheda/wordpress-develop/blob/ticket/59745/src/wp-includes/media.php#L1524

Reply:
@mukeshpanchal27 Let me know if any changes are required in this PR